## SMRTest Microbe Grant Program

* Aaron Brooks
* 24.06.2015

Annaluru et al recently described a designer yeast strain with a fully synthetic chromosome [Annaluru et al 2014]. One of the fascinating features of this strain is that it harbors loxPsym sites flanking every gene before the 3’-UTR. This allows "scrambl”ing of its genome. Our lab is interested in the generation and phenotypic effects of transcriptional heterogeneity, particularly the generation of alternate transcript isoforms. We have characterized extensive transcriptional heterogeneity in yeast using novel a sequencing approach (TIF-seq) [Palechano et al 2013]. We would like to apply an improved version of this protocol to scrambled yeast strains with increased fitness in defined environments. To do so, however, we will need to characterize the precise genetic re-arrangements in these strains genome-wide. Currently, this is impractical with short read technologies, since very few of the reads will be indicative of re-arrangement. The PacBio system is ideal for this application because long reads can correctly identify contiguous regions of the genome that have been re-arranged. Combining PacBio re-annotation of scramble yeast 2.0 genomes with TIF-seq measurement of transcriptional heterogeneity will provide a platform to discover how genetic factors contribute to isoform generation and how these isoforms contribute to fitness.  